angular-workspace/projects/ni/nimble-angular/label-provider/core/nimble-label-provider-core-with-defaults.directive.ts

```diff
@@ -15,5 +15,10 @@ export class NimbleLabelProviderCoreWithDefaultsDirective {
         this.elementRef.nativeElement.popupDismiss = $localize`:Nimble popup - dismiss|:Close`;
         this.elementRef.nativeElement.numericDecrement = $localize`:Nimble numeric - decrement|:Decrement`;
         this.elementRef.nativeElement.numericIncrement = $localize`:Nimble numeric - increment|:Increment`;
+        this.elementRef.nativeElement.popupIconError = $localize`:Nimble icon - error|:Error`;
+        this.elementRef.nativeElement.popupIconWarning = $localize`:Nimble icon - warning|:Warning`;
+        this.elementRef.nativeElement.popupIconInformation = $localize`:Nimble icon - information|:Information`;
```

Comment:
based on the other examples
```suggestion
        this.elementRef.nativeElement.popupIconError = $localize`:Nimble popup icon - error|:Error`;
        this.elementRef.nativeElement.popupIconWarning = $localize`:Nimble popup icon - warning|:Warning`;
        this.elementRef.nativeElement.popupIconInformation = $localize`:Nimble popup icon - information|:Information`;
```